Bottom: 9a6fcc3160ceee211472b7aaee9215b040985bd1
Top:    92169934e9702bee338423727927332d1ed0ee7f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 22:09:27 +0100

hid/gtk: Enable transparent lines in GL renderer


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 16247a8..385351e 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -79,7 +79,7 @@ ghid_set_layer (const char *name, int group, int empty)
 
   if (idx >= 0 && idx < max_copper_layer + 2)
     {
-      priv->trans_lines = false;
+      priv->trans_lines = true;
       return PCB->Data->Layer[idx].On;
     }
   if (idx < 0)
